packages/clerk-js/src/ui/elements/PhoneInput/PhoneInput.tsx

```diff
@@ -124,9 +121,12 @@ const PhoneInputBase = (props: PhoneInputProps) => {
         />
       </Select>
       <Input
-        value={formattedPhoneNumber}
+        value={formattedNumber}
         onPaste={handlePaste}
         onChange={handlePhoneNumberChange}
+        onInput={e => {
+          console.log('input', e);
```

Comment:
@nikosdouvlis you probably forgot this console.log

Reply:
Fixing 😵‍💫